lib/services/packagist.rb

```diff
@@ -37,14 +36,13 @@ def domain
 
   def full_domain
     if data['domain'].to_s == ''
-      'http://packagist.org'
+      'https://packagist.org'
     else
       data['domain']
-    end.lstrip.sub(/[\/\s]+$/,'')
+    end.lstrip.sub(/[\/\s]+$/,'').sub(/^http:\/\/packagist.org/, 'https://packagist.org')
```

Comment:
Can you use `\z` to denote the end of the string instead of `$`? 

Can you also use `\A` to denote the beginning of the string instead of the `^` here?

Thank you!